_posts/2019-08-13-how-to-deploy-production-grade-vpc-aws.adoc

```diff
@@ -638,18 +638,18 @@ AWS accounts::
 
 // TODO: we should be embedding code snippets from real, working code intended for production (i.e., not the stuff
 // under the examples folder of each repo). Perhaps this guide should live with one of the Acme repos for now so we can
-// embed examples from our Ref Arch code?
+// embed examples from our Ref Arch code? (Rho:) Yes.
```

Comment:
Agreed with you, but we prob don't need to check-in this update 😁

Reply:
Right. I will remove this update! I think we should hold off on the embedded examples in the interest of effort time and other things being higher priority.